content/en/change_tracking/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Change Tracking
+further_reading:
+- link: "/monitors/status/"
+  tag: "Documentation"
+  text: "Monitor Status Page"
+- link: "/dashboards/"
+  tag: "Documentation"
+  text: "Dashboards"
+- link: "/tracing/services/service_page/"
+  tag: "Documentation"
+  text: "Service Page"
+---
+
+## Overview
+
+Change Tracking streamlines troubleshooting and incident response by surfacing relevant changes to your service and its dependencies, enabling faster detection and remediation when issues arise.
+
+{{< img src="/change_tracking/change-tracking-overview.png" alt="The details of a change on the Recent Changes timeline in the Service Summary" style="width:100%;" >}}
+
+Change Tracking supports monitoring of a range of modifications to your service and its dependencies including:
+- Deployments
+- Feature Flags
+- Database Modifications
+- Schema Changes
+- Scale Adjustments
+- Kubernetes Adjustments
+- Kubernetes Pod Crashes
+- Watchdog Alerts
+
+For details on specific types of supported changes and setup requirements [see below](#tracked-changes).
+
+## Prerequisites
+
+To use Change Tracking, ensure the following:
+
+- **Monitors**: Confirm the service is specified in the monitor query, selected as part of a group, or set the `service` tag.
```

Comment:
Can we split the pre requisite in individual sections? I think it would be easier to read